File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int n = nums.size();
        vector <int> res;
        for (int i = 0; i < n; i++){
            if (nums[i] != val) res.push_back(nums[i]);
        }
        int k = res.size();
        for (int j = 0; j < k; j++) nums[j] = res[j];
        return k;
    }
};

File: Data Structures & Algorithms/majority-element/submission-0.java
class Solution {
    public int majorityElement(int[] nums) {
     int candicate = 0;
     int cnt = 0;
        for (int num : nums){
            if (cnt == 0) candicate = num;
            if(candicate != num) cnt--;
            else cnt++;
        }
        return candicate;
        
    }
}